src/utils.js

```diff
@@ -30,4 +30,35 @@ function getNodeProperty(node, path) {
   return property;
 }
 
-module.exports = getNodeProperty;
+/**
+ * Check if the file path follows an Embroider build pattern
+ * @param {string} filepath
+ * @returns boolean
+ */
+function hasEmbroiderPrefix(filepath) {
+  if (typeof filepath !== 'string') return;
+
+  const separator = filepath.includes('\\') ? '\\' : '/';
+  return filepath.split(separator).includes('embroider');
+}
+
+/**
+ * Get a file path with the Embroider prefix segments stripped out
+ * @param {string} filepath  E.g. /private/var/folders/abcdefg1234/T/embroider/098765/tests/acceptance/my-test.js
+ * @returns {string} E.g. tests/acceptance/my-test.js
+ */
+function getEmbroiderStrippedPrefixPath(filepath) {
+  if (typeof filepath !== 'string') return;
+
+  const separator = filepath.includes('\\') ? '\\' : '/';
```

Comment:
Why do you need this? If you're trying to get a platform specific path, you can use [path.sep](https://nodejs.org/api/path.html#path_path_sep).